eksconfig/cluster-loader.go

```diff
@@ -28,32 +33,8 @@ func (spec *ClusterLoaderSpec) Default(cfg *Config) {
 	}
 	if len(spec.TestConfigUris) == 0 {
 		spec.TestConfigUris = []string{
```

Comment:
Does this default make sense?

Reply:
Default makes sense to me. It's a very basic test.